src/app/pdf-viewer-dialog/pdf-viewer-dialog.component.css

```diff
@@ -0,0 +1,13 @@
+.mat-dialog-content {
+    padding: 10px !important;
+}
+
+.container {
+    padding: 32px;
+}
+
+.download-btn {
+    padding: 8px;
+    width: 40%;
+    margin: 16px 4px 16px 4px;
+}
```

Comment:
Styling on first page of pdf is missing. Fix that .

![image](https://github.com/rajeev1202/CRM-frontend/assets/29862832/cdec6f01-f38b-4703-a609-a11184895241)

![image](https://github.com/rajeev1202/CRM-frontend/assets/29862832/907e9ecb-e016-4089-83b7-f7c7f3e77faf)